### PR TITLE
[SPG-203]: adding env group to harness go sdk

### PR DESCRIPTION
**Pull request by RajBaviskar**
- adding env group to harness go sdk

### CODE REVIEW

**Review by rathodmeetsatish — COMMENTED** [review]
please remove swagger.yaml file changes

File: harness/nextgen/model_environment_group_response.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/*
+ * Harness NextGen Software Delivery Platform API Reference
+ *
+ * This is the Open Api Spec 3 for the NextGen Manager. This is under active development. Beware of the breaking change with respect to the generated code stub
+ *
+ * API version: 3.0
+ * Contact: contact@harness.io
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+package swagger
```
Comment (rathodmeetsatish) [inline harness/nextgen/model_environment_group_response.go:10]: package should be nextgen for all the files

Comment (RajBaviskar) [reply harness/nextgen/model_environment_group_response.go]: fixed it.

**Comment by rathodmeetsatish** [comment]
> please remove swagger.yaml file changes

@RajBaviskar i meant restore the file dont change anything in it

**Review by rijajoo — APPROVED** [review]

**Review by rathodmeetsatish — APPROVED** [review]

**Pull request merged**